src/message/message.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { CreateMessageDto } from './dto/create-message.dto';
+import { UpdateMessageDto } from './dto/update-message.dto';
+
+@Injectable()
+export class MessageService {
```

Comment:
- This is a boiler plate code. 
- Messages service should be implemented correctly 
- use prisma to fetch data from DB

Reply:
Yes, Biny, it's not done yet. I didn't mean to send a PR (newbie mistake).